### PR TITLE
update grunttype.json. thx pmsf/leevo

### PR DESCRIPTION
**Pull request by plinytheelder**
bringing over grunttype updates from pmsf

## Description

updating grunttype 
## Motivation and Context

because it changes
## How Has This Been Tested?

nope, but seems fine
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/util/grunt_types.json [hunk 1]
```diff
@@ -1,291 +1,258 @@
...
-        "first": ["1","4","7"],
-        "second":["2","5","8"],
-        "third":["3","6","9"]
+        "first": ["001_00","004_00","007_00"],
+        "second":["002_00","005_00","008_00"],
+        "third":["003_00","006_00","009_00"]
     }
   },
   "5": {
-    "type": "Mixed",
```
Comment (fosJoddie) [inline src/util/grunt_types.json]: Poracle needs to call these mixed to  be able to track them

File: src/util/grunt_types.json [hunk 2]
```diff
@@ -1,291 +1,258 @@
...
+    "grunt": "Candela"
   },
   "3": {
     "type": "",
-    "grunt": "Spark",
-    "gender": 1
+    "grunt": "Spark"
   },
   "4": {
-    "type": "Mixed",
```
Comment (fosJoddie) [inline src/util/grunt_types.json]: Poracle needs to call these mixed to  be able to track them

File: src/util/grunt_types.json [hunk 3]
```diff
@@ -1,291 +1,258 @@
...
+    "type": "",
     "grunt": "Male Grunt",
-    "gender": 1,
-    "second_reward": true,
+    "second_reward": "true",
     "encounters":{
-        "first": ["1","4","7"],
-        "second":["2","5","8"],
-        "third":["3","6","9"]
+        "first": ["001_00","004_00","007_00"],
```
Comment (fosJoddie) [inline src/util/grunt_types.json]: Search replace _00 with blank text, Poracle knows not about forms gere I think?

File: src/util/grunt_types.json (hunk 1 above)
Comment (plinytheelder) [reply src/util/grunt_types.json]: oops

**Pull request closed without merging**